js/background.js

```diff
@@ -811,6 +828,52 @@
       }
     });
 
+    Whisper.events.on('inviteFriends', async groupConvo => {
+      if (appView) {
+        appView.showInviteFriendsDialog(groupConvo);
+      }
+    });
+
+    Whisper.events.on(
+      'invitationAccepted',
```

Comment:
`invitation` sounds like it could be vague, and not super suited for extending to other types of invitations (whatever that might be in the future).
Maybe a more specific name like `publicChatInvitationAccepted` would be better?

Reply:
Updated.